[all] (Host-specific APIs) Create a host-specific API model page (#2046)
OfficeDev/office-js-docs-pr: docs/includes/office-js-api-models.md

    > [!NOTE]
    > There is also a host-specific API for [Visio](../reference/overview/visio-javascript-reference-overview.md), but you can use it only in SharePoint Online pages to interact with Visio diagrams that have been embedded in the page. Office web add-ins are not supported in Visio.

>     Learn more about how to use this API model in the article [Using the host-specific API model](../develop/host-specific-api-model.md).

Reviewer: ```suggestion
    See [Using the host-specific API model](../develop/host-specific-api-model.md) to learn more about this API model .
```
